config/locales/en.yml

```diff
@@ -1124,10 +1124,10 @@ en:
   login_password: "Password:"
   remember_user: "Remember me:"
   signin:
-    button1: "MUSC Users Click Here"
```

Comment:
Please remove en.yml from your pull request. We need to keep our MUSC wording in place.